lib/stack_master/validator.rb

```diff
@@ -13,6 +13,7 @@ def perform
     rescue Aws::CloudFormation::Errors::ValidationError => e
       $stderr.puts "Validation Failed"
       $stderr.puts e.message
+      false
```

Comment:
Yeah but the Command instance is not currently available from the CLI method, so how do we retrieve the return value?